stdlib/printexc.mli

```diff
@@ -22,6 +22,10 @@ val to_string: exn -> string
 (** [Printexc.to_string e] returns a string representation of
    the exception [e]. *)
 
+val to_string_default: exn -> string
+(** [Printexc.to_string_opt e] returns a string that would be generated
+    without registered printers of exception [e] *)
```

Comment:
- `s/to_string_opt/to_string_default`
- wording suggestion: `[Printexc.to_string_default e] returns a string representation of the exception [e], ignoring all registered exception printers.`.

Reply:
It also needs `@since 4.09`